Minio/Minio.csproj

```diff
@@ -4,7 +4,7 @@
     <LangVersion>latest</LangVersion>
     <AssemblyName>Minio</AssemblyName>
     <RootNamespace>Minio</RootNamespace>
-    <TargetFrameworks>netstandard2.0;net46;net5.0;</TargetFrameworks>
+    <TargetFrameworks>net5.0; net6.0</TargetFrameworks>
```

Comment:
Why netstandard2.0 has been removed? All legacy applications will not be able to get the latest fixes. There are a lot of apps built with netstandard2.0 and support that very important support them as well. You can take a look at my [repo](https://github.com/ramonesz297/minio-client) and some performance improvements for net5 and net6 can be used here without removing netstandard2.0 support.

Reply:
netstandard is not supported anymore by MSFT https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#cadence

Reply:
@harshavardhana  Are you going to completely remove net4.x support?

Reply:
Yes